### PR TITLE
feat(ua): Add custom UA suffix

### PR DESCRIPTION
**Pull request by julienbourdeau**

### CODE REVIEW

File: src/ServiceProvider.php [hunk 1]
```diff
@@ -9,6 +9,8 @@ class ServiceProvider extends LaravelServiceProvider
 {
     public function boot()
     {
+        AlgoliaUserAgent::addSuffixUserAgentSegment('; Laravel Scout macros package', '0.x');
```
Comment (Haroenv) [inline src/ServiceProvider.php:12]: Shouldn't this be a prefix?

Comment (julienbourdeau) [reply src/ServiceProvider.php:12]: I thought it was from the most important to the less important. Is there a special order?

Comment (Haroenv) [reply src/ServiceProvider.php:12]: We should get that in the spec, I thought it was a dependency chain (macros; ...; php)

Comment (julienbourdeau) [reply src/ServiceProvider.php:12]: ¯\_(ツ)_/¯ haha

**Review by rayrutjes — CHANGES_REQUESTED** [review]
Can we add some tests maybe?

File: src/ServiceProvider.php (hunk 1 above)
Comment (julienbourdeau) [reply src/ServiceProvider.php:12]: Actually your way would make more sense. As soon as you read "scout settings integration". You know scout and php client are used underneath.

**Comment by julienbourdeau** [comment]
I will add tests as soon as possible, I'll merge this one for now. I created https://github.com/algolia/laravel-scout-algolia-macros/issues/16 for the test suite.

**Pull request merged**